packages/wallets/src/components/AccountsList/AccountsList.tsx

```diff
@@ -16,6 +17,10 @@ const AccountsList: FC<TProps> = ({ accountsActiveTabIndex, onTabClickHandler })
     const { isDesktop } = useDevice();
     const { localize } = useTranslations();
     const { data: isEuRegion, isLoading: isEuRegionLoading } = useIsEuRegion();
+    const { data: activeWallet } = useActiveWalletAccount();
+    const [isP2PWalletEnabled, isGBLoaded] = useGrowthbookIsOn({
+        featureFlag: 'p2p_wallet_enabled',
+    });
```

Comment:
Added `p2p_wallet_enabled` feature flag as a dependency to ensure card does not get blocked by wallets p2p release flow